server/middleware/security/authentication.js

```diff
@@ -220,6 +221,35 @@ export default function (app) {
       }
     },
 
+    authenticateInternalUserByJwt() {
+      return (req, res, next) => {
+        this.parseJwtNoExpiryCheck(req, res, (e) => {
+          if (e) {
+            return next(e);
+          }
+          this.checkJwtExpiry(req, res, (e) => {
+            if (e) {
+              return next(e);
+            }
+            this._authenticateUserByJwt(req, res, (e) => {
+              if (e) {
+                return next(e);
+              }
+              this._authenticateInternalUserById(req, res, next);
+            });
+          });
+        });
+      }
+    },
+
+    _authenticateInternalUserById: (req, res, next) => {
+      if (_.contains([1,2,8,30,40,212,772], req.jwtPayload.sub)) {
```

Comment:
could be extracted to constants if wished (IDs are also in weekly report script)